Lib/test/test_urllib2.py

```diff
@@ -839,7 +838,7 @@ def test_file(self):
             try:
                 h.file_open(req)
             # XXXX remove OSError when bug fixed
```

Comment:
It's removed now: https://github.com/python/cpython/pull/1020 
Thank you for catching.